content/en/docs/Tasks/performance-profiling.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Continuous Profiling"
+weight: 10
+description: >
+  The goal of this document is to familiarize you with OLM's stance on performance profiling.
+---
+
+## Prerequisites 
+
+- [go](https://golang.org/dl/)
+
+## Background
+
+OLM utilizes the [pprof package](https://golang.org/pkg/net/http/pprof/) from the standard go library to expose performance profiles for the OLM Operator, the Catalog Operator, and Registry Servers. Due to the sensitive nature of this data, client requests against the pprof endpoint are rejected unless they are made with the certificate data kept in the `pprof-cert secret` in the `olm namespace`. 
+Kubernetes does not provide a native way to prevent pods on cluster from iterating over the list of available ports and retrieving the data exposed. Without authetnicating the requests, OLM could leak customer usage statistics on multitenant clusters. If the aforementioned secret does not exist the pprof data will not be accessable.
```

Comment:
Just a quick driveby as I see this is still in a draft:

authetnicating -> authenticating